### PR TITLE
Update help window with current commands

### LINKED ISSUE #263: Help window is not updated with all new commands added

### PR DESCRIPTION
**Pull request by joeltio**
Closes #263

### CODE REVIEW

File: src/main/java/bizbook/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -9,8 +9,11 @@
  * Clears the address book.
  */
 public class ClearCommand extends Command {
-
     public static final String COMMAND_WORD = "clear";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Undos the previous command that was executed.\n"
```
Comment (sheenkerr) [inline src/main/java/bizbook/logic/commands/ClearCommand.java:14]: I believe this is an error

**Review by sheenkerr — CHANGES_REQUESTED** [review]

File: src/main/java/bizbook/ui/CommandTablePanel.java
Comment (Nicholas-Cheng-De-Fei) [inline src/main/java/bizbook/ui/CommandTablePanel.java:1]: I believe can pull from master and add in the `deletetag` command as well

**Review by Nicholas-Cheng-De-Fei — CHANGES_REQUESTED** [review]
Maybe for convenience, as what you had mentioned about the `deleteTag` command, I believe the following commands, their message usage can to be updated as they include the `[]`
- `editNote`
- `deleteNote`
- `addNote`

Since the fields are mandatory for the function to execute.

image

**Review by Nicholas-Cheng-De-Fei — APPROVED** [review]
LGTM

**Review by sheenkerr — APPROVED** [review]
LGTM

**Pull request merged**